### ISSUE TITLE
Need Translations for oUF NeavRaid

### ISSUE THREAD

**Issue by Grimsbain** [lithammer/NeavUI#264]
We need translations for the oUF_NeavRaid config. If you want to submit a translation for any of the following languages just copy the text below and replace the text in quotation marks with appropriately translated text. When translating don't worry about literal translations. Chose translations that would be natural to native speakers of that language but do keep in mind space restrictions.

- [ ] Brazilian Portuguese
- [ ] French
- [ ] German
- [ ] Italian
- [ ] Korean
- [ ] Latin American Spanish
- [ ] Russian
- [ ] Simplified Chinese
- [ ] Spanish
- [ ] Traditional Chinese

```
L.GeneralOptions = "General Options"
L.ShowSolo = "Show Solo"
L.ShowParty = "Show Party"
L.AssistFrame = "Show Main Tank & Assist Frames"
L.SortByRole = "Sort By Role"
L.SortByRoleTooltip = "Sorts the raid frames by role instead of group."
L.ShowRoleIcons = "Show Role Icons"
L.AnchorToControls = "Unlock Frame"
L.HorizontalHealthBars = "Horizontal Health Bars"
L.ShowPowerBars = "Show Power Bars"
L.ManaPowerBarsOnly = "Mana Only"
L.HorizontalPowerBars = "Horizontal Power Bars"
L.AuraOptions = "Buff & Debuff Options"
L.IndicatorSizeSlider = "Indicator Size"
L.DebuffSizeSlider = "Debuff Size"
L.LayoutOptions = "Display Options"
L.Orientation = "Orientation"
L.InitialAnchor = "Initial Anchor"
L.NameLengthSlider = "Name Length"
L.FrameWidthSlider = "Frame Width"
L.FrameHeightSlider = "Frame Height"
L.FrameOffsetSlider = "Frame Offset"
L.FrameScaleSlider = "Frame Scale"

L.Vertical = "Vertical"
L.Horizontal = "Horizontal"
L.TopLeft = "Top Left"
L.BottomLeft = "Bottom Left"
L.TopRight = "Top Right"
L.BottomRight = "Bottom Right"
```

**Comment by Grimsbain** [comment]
Well also need translations for nCore.

```
L.OptionsLabel = "Options"
L.AltBuy = "Alt Buy"
L.AltBuyTooltip = "Hold the alt key to buy a stack of an item."
L.AltBuyVendorToolip = "<Alt-click, to buy an stack>"
L.ArchaeologyHelper = "Archaeology Helper"
L.ArchaeologyHelperTooltip = "Double right click to survey at digsites."
L.AutoGreed = "Auto Greed"
L.AutoGreedTooltip = "Automaticly role greed on green items at max level."
L.AutoQuest = "Auto Quest"
L.AutoQuestTooltip = "Hold shift to automaticly accept and turnin quests."
L.CombatWarning = "nCore: You cant do this in combat!"
L.Dressroom = "Dressroom Tweaks"
L.DressroomTooltip = "Adds clothed and naked buttons to the dressroom frame."
L.Durability = "Durability"
L.DurabilityTooltip = "Adds durability information to the character frame."
L.ErrorFilter = "Error Filter"
L.ErrorFilterTooltip = "Filter common error spam."
L.Fonts = "Font Fix"
L.FontsTooltip = "Modifies some default font objects."
L.ObjectiveTracker = "Moveable Objective Tracker"
L.ObjectiveTrackerTooltip = "Makes the objective tracker movable."
L.ObjectiveTrackerButtonTooltip = "Shift + left-click to Drag"
L.MapCoords = "Map Coords"
L.MapCoordsTooltip = "Adds player and cursor location to the world map."
L.MoveTalkingHeads = "Moveable Talking Heads"
L.MoveTalkingHeadsTooltip = "Makes the talking heads frame movable. /alertframemover to unlock the frame."
L.QuestTracker = "Quest Tracker"
L.QuestTrackerTooltip = "Add current number of quests to the worldmap title text."
L.Skins = "Skin Addons"
L.SkinsTooltip = "Skin Omen, DBM, TinyDPS, Recount, Skada, and Numeration to closer match the Neav UI look."
L.SpellID = "Spell IDs"
L.SpellIDTooltip = "Add spell ids to the tooltip."
L.VignetteAlert = "Vignette Alerts"
L.VignetteAlertTooltip = "Alerts you when you detect a rare or chest."
```

**Comment by Rainrider** [comment]
German:

```
L.GeneralOptions = "Allgemeine Einstellungen"
L.ShowSolo = "Anzeigen wenn solo"
L.ShowParty = "Gruppe anzeigen"
L.AssistFrame = "Haupttank und -assistent anzeigen" (COMPACT_UNIT_FRAME_PROFILE_DISPLAYMAINTANKANDASSIST)
L.SortByRole = "Nach Rolle sortieren"
L.SortByRoleTooltip = "Sortiert Schlachtzugsmitglieder nach Rolle anstatt nach Gruppe."
L.ShowRoleIcons = "Rollensymbole anzeigen"
L.AnchorToControls = "Freischalten" (UNLOCK)
L.HorizontalHealthBars = "Horizontale Gesundheitsbalken"
L.ShowPowerBars = "Ressourcenbalken anzeigen" (COMPACT_UNIT_FRAME_PROFILE_DISPLAYPOWERBAR)
L.ManaPowerBarsOnly = "Nur Mana"
L.HorizontalPowerBars = "Horizontale Ressourcenbalken"
L.AuraOptions = "Einstellungen für Stärkungs- und Schwächungszauber"
L.IndicatorSizeSlider = "Indikatorgröße"
L.DebuffSizeSlider = "Schwächungszaubergröße"
L.LayoutOptions = "Anzeigeeinstellungen"
L.Orientation = "Ausrichtung"
L.InitialAnchor = "Initialanker"
L.NameLengthSlider = "Namenslänge"
L.FrameWidthSlider = "Breite"
L.FrameHeightSlider = "Höhe"
L.FrameOffsetSlider = "Abstand"
L.FrameScaleSlider = "Skallierung"

L.Vertical = "Vertikal"
L.Horizontal = "Horizontal"
L.TopLeft = "Oben links"
L.BottomLeft = "Unten links"
L.TopRight = "Oben rechts"
L.BottomRight = "Unten rechts"
```
```
L.OptionsLabel = "Einstellungen"
L.AltBuy = "Alt-Kaufen"
L.AltBuyTooltip = "Die Alt-Taste gedrückt halten um einen Stapel von einem Gegenstand zu kaufen."
L.AltBuyVendorToolip = "<Alt-Klick um einen Stapel zu kaufen>"
L.ArchaeologyHelper = "Archäologie-Gehilfe"
L.ArchaeologyHelperTooltip = "Doppelt rechtsklicken um Grabungsstätten zu untersuchen."
L.AutoGreed = "Autom. Gier"
L.AutoGreedTooltip = "Automatisch Gier bei grünen Gegenständen würfeln, wenn auf Höchststufe."
L.AutoQuest = "Autom. Quests"
L.AutoQuestTooltip = "Die Umschalttaste gedrückt halten um Quests automatisch anzunehmen und abzuschliessen."
L.CombatWarning = "nCore: Das ist im Kampf nicht möglich!"
L.Dressroom = "Anpassungen für die Anprobe"
L.DressroomTooltip = "Fügt dem Anprobefester Buttons zum An- und Ausziehen hinzu."
L.Durability = "Haltbarkeit" (DURABILITY)
L.DurabilityTooltip = "Zeigt Haltbarkeitinformationen im Charakterfenster an."
L.ErrorFilter = "Fehlerfilter"
L.ErrorFilterTooltip = "Filtert oft vorkommende Fehlermeldungen aus."
L.Fonts = "Schriftenreparatur"
L.FontsTooltip = "Modifiziert manche der Standardschriften."
L.ObjectiveTracker = "Verstellbares Fortschrittsfenster"
L.ObjectiveTrackerTooltip = "Macht das Fortschrittsfenster verstellbar."
L.ObjectiveTrackerButtonTooltip = "Umschalttaste + Linksklick zum Ziehen"
L.MapCoords = "Kartenkoordinaten"
L.MapCoordsTooltip = "Zeigt die Player- und die Kursorposition auf der Weltkarte an."
L.MoveTalkingHeads = "Verstellbarer Talking Heads" @TODO
L.MoveTalkingHeadsTooltip = "Macht das talking heads Fenster verstellbar. /alertframemover um das Fenster freizuschalten." @TODO
L.QuestTracker = "Quest Tracker" @TODO
L.QuestTrackerTooltip = "Zeigt die Anzahl angenommener Quests in der Titelleiste der Weltkarte an."
L.Skins = "Addonsdarstellung"
L.SkinsTooltip = "Das Aussehen von Omen, DBM, TinyDPS, Recount, Skada und Numeration anpassen, damit sie dem Aussehen von Neav UI ähneln."
L.SpellID = "Zauber IDs"
L.SpellIDTooltip = "Zauber IDs in Tooltips anzeigen."
L.VignetteAlert = "Alarmierungen für Vignetten"
L.VignetteAlertTooltip = "Löst ein Alarm aus, wenn seltene Kreaturen oder Truhen in der Nähe sind."
```

Some notes about the translations:
- Tried to use the wording used in the default UI as much and as closely as possible (with a few exceptions listed below) and provided some [global strings](https://www.townlong-yak.com/framexml/live/GlobalStrings.lua/DE) that could be used instead.
- Tried to accommodate to language purists except where "English" Terms are explicitly used in the default UI (like Tooltips and Buttons). Particularly tried to avoid the use of the word "Frame", so "Frame Width" became just "Width" and "Unlock Frames" - "Unlock" (should be ok as the context makes it clear).
- Tried to avoid addressing the user directly. "nCore: You cant do this in combat!" became "nCore: This is not possible in combat!" The reasoning is that in German the formal and informal forms of addressing someone differ and that in-game an old formal form is used (2nd person plural rather than 3rd person plural, which is used nowadays). Since the english "you" does not convey which form would be preferred I opted for this.
- I don't have proper translations for "talking heads" and "Quest Tracker". The former would be "Sprechende Köpfe", which sounds funny and the latter could be ok as it is. Look for @TODO labels in the translations above.
- I am not a native German speaker
- I don't use Neav UI (just nChat)

**TL;DR;**
Please let someone review this :)

**Comment by Leiludallas** [comment]
Hi

French translation for my favorite legion ui.
Actually, I don't play wow, I can't clearly see each line details.

L.GeneralOptions = "Options Générales"
L.ShowSolo = "Afficher Solo"
L.ShowParty = "Afficher Groupe"
L.SortByRole = "Trier Par Rôle"
L.SortByRoleTooltip = "Trier les cadres de raid par rôle et non par groupe."
L.ShowRoleIcons = "Afficher les icônes de rôle"
L.HorizontalHealthBars = "Barres de vie Horizontales"
L.ManaPowerBarsOnly = "Seulement les barres de mana"
L.HorizontalPowerBars = "Barres de Puissance Horizontales"
L.AuraOptions = "Options des Améliorations et Affaiblissements"
L.IndicatorSizeSlider = "Taille de l'indicateur"
L.DebuffSizeSlider = "Taille des affaiblissements"
L.LayoutOptions = "Options d'affichage"
L.Orientation = "Orientation"
L.InitialAnchor = "Ancre initiale"
L.StatusBarTexture = "Texture de la barre de statut"
L.Font = "Police"
L.FontSize = "Taille de la Police"
L.NameLengthSlider = "Longueur des Noms"
L.FrameWidthSlider = "Largeur"
L.FrameHeightSlider = "Hauteur"
L.FrameOffsetSlider = "Décalage"
L.FrameScaleSlider = "Taille"

L.Vertical = "Vertical"
L.Horizontal = "Horizontal"
L.TopLeft = "Haut Gauche"
L.BottomLeft = "Bas Gauche"
L.TopRight = "Haut Droit"
L.BottomRight = "Bas Droite"

**Comment by Insopitus** [comment]
Simplified Chinese

> L.GeneralOptions = "常规设定"
> L.ShowSolo = "单人显示"
> L.ShowParty = "小队显示"
> L.SortByRole = "按职责排序"
> L.SortByRoleTooltip = "按职责排序角色而非队伍。"
> L.ShowRoleIcons = "显示职责图标"
> L.HorizontalHealthBars = "水平血条"
> L.ManaPowerBarsOnly = "只显示法力条"
> L.HorizontalPowerBars = "水平能量条"
> L.AuraOptions = "增减益设定"
> L.IndicatorSizeSlider = "指示器大小"
> L.DebuffSizeSlider = "减益尺寸"
> L.LayoutOptions = "显示设定"
> L.Orientation = "定位"
> L.InitialAnchor = "初始锚点"
> L.StatusBarTexture = "状态条材质"
> L.Font = "字体"
> L.FontSize = "字号"
> L.NameLengthSlider = "名称长度"
> L.FrameWidthSlider = "宽度"
> L.FrameHeightSlider = "高度"
> L.FrameOffsetSlider = "偏移"
> L.FrameScaleSlider = "比例"
> 
> L.Vertical = "竖直"
> L.Horizontal = "水平"
> L.TopLeft = "左上"
> L.BottomLeft = "左下"
> L.TopRight = "右上"
> L.BottomRight = "右下"

> L.OptionsLabel = "选项"
> L.AltBuy = "Alt Buy"
> L.AltBuyTooltip = "按住Alt以购买一组物品."
> L.AltBuyVendorToolip = "<Alt+点击，购买一组>"
> L.ArchaeologyHelper = "考古助手"
> L.ArchaeologyHelperTooltip = "双击右键在挖掘点勘测."
> L.AutoGreed = "自动贪婪"
> L.AutoGreedTooltip = "自动对满级绿装roll贪婪."
> L.AutoQuest = "自动任务"
> L.AutoQuestTooltip = "按住shift自动交接任务."
> L.CombatWarning = "nCore: 你无法在战斗中这么做!"
> L.Dressroom = "试衣间调整"
> L.DressroomTooltip = "给试衣间框体增加了裸体和着装按钮."
> L.Durability = "耐久"
> L.DurabilityTooltip = "给角色框体增加了耐久显示."
> L.ErrorFilter = "错误过滤"
> L.ErrorFilterTooltip = "过滤常见错误信息."
> L.Fonts = "字体修复"
> L.FontsTooltip = "修改了一些默认字体目标."
> L.ObjectiveTracker = "可移动的目标追踪"
> L.ObjectiveTrackerTooltip = "让目标追踪可移动."
> L.ObjectiveTrackerButtonTooltip = "Shift + 左键单击来移动"
> L.MapCoords = "地图坐标"
> L.MapCoordsTooltip = "在世界地图上添加了玩家和光标的坐标位置."
> L.MoveTalkingHeads = "可移动的任务头像框"
> L.MoveTalkingHeadsTooltip = "让任务中“说话的头像”框体可以移动. 输入/alertframemover 来解除框体锁定."
> L.QuestTracker = "任务追踪"
> L.QuestTrackerTooltip = "在世界地图的标题文字上添加正在进行的任务数量."
> L.Skins = "皮肤插件"
> L.SkinsTooltip = "对 Omen, DBM, TinyDPS, Recount, Skada, 与 Numeration进行了皮肤替换使其更接近Neav UI风格."
> L.SpellID = "法术ID"
> L.SpellIDTooltip = "在鼠标信息框显示法术ID."
> L.VignetteAlert = "宝箱警报"
> L.VignetteAlertTooltip = "当你发现稀有怪物或宝箱时警报."
